### DISCUSSION TITLE
Apollo Save Tool v0.6.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps2#5] [Announcements]
First public release.

### Added

* Save-game listing (from MC, USB, CD, Host)
* Import saves (in PSU, PSV, MAX, CBS, XPS, SPS formats) to PS2 memcards
* Export and backup memcard saves (to PSU, PSV, CBS, ZIP formats)
* Save-game patching (with SaveWizard / GameGenie codes or BSD scripts)
* Save-data hex editing

### Notes

* Limitations
  * No network features
  * No PS1 save-game support

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps2/releases/tag/v0.6.0'>Apollo Save Tool v0.6.0</a>.</em>

**Comment by whiteShark881** [comment]
Not working on PS2 slim 90xxx series with Matrix chip, when starting from black
